src/wgpu_sensor.cc

```diff
@@ -294,19 +365,101 @@ namespace wgpu_sensor
 
       set_transforms(rt_scene, rt_runtime, rt_scene_update);
 
+      // Old camera rendering code
+      /*
       auto camera_pose = gz::sim::worldPose(camera_entity, _ecm);
       auto view_matrix = create_view_matrix(camera_pose.Pos().X(), camera_pose.Pos().Y(), camera_pose.Pos().Z(), camera_pose.Rot().X(), camera_pose.Rot().Y(), camera_pose.Rot().Z(), camera_pose.Rot().W());
       render_depth(rt_depth_camera, rt_scene, rt_runtime, view_matrix);
       free_rt_scene_update(rt_scene_update);
       free_view_matrix(view_matrix);
       high_resolution_clock::time_point t2 = high_resolution_clock::now();
       auto duration = std::chrono::duration_cast<std::chrono::milliseconds>(t2 - t1).count();
-      gzmsg << "Time taken to render depth: " << duration << std::endl;
+      gzmsg << "Time taken to render depth: " << duration << std::endl; */
+
+      for (auto const& [entityId, sensor] : this->entitySensorMap)
```

Comment:
Move this out of the lambda closure we don't need to update this every single time we update an object.